common/include/pcl/common/random.h

```diff
@@ -68,6 +68,38 @@ namespace pcl
       using type = std::normal_distribution<T>;
     };
 
+    template <typename RandomEngineT = std::default_random_engine>
+    class RandomBase {
+    public:
+      using SeedT = typename RandomEngineT::result_type;
+
+      RandomBase(SeedT seed = std::random_device()()) : seed_(seed), rng_(seed) {}
```

Comment:
Changed brackets to make it more obvious what's going on
```suggestion
      RandomBase(SeedT seed = std::random_device{}()) : seed_(seed), rng_(seed) {}
```